### ISSUE TITLE
Fonts (and only fonts) are getting cloudflared.

### ISSUE THREAD

**Issue by herp-a-derp** [yiffparty/tracker#15]
![2017-08-19 00_54_11-w7-1 rdp session - 10 1 1 21 - remote desktop connection](https://user-images.githubusercontent.com/5681826/29484713-49c595ba-8479-11e7-88a6-a86123da9bf6.png)

I'm not sure why, but when I access the site, it seems like the font-files are getting "protected" by the cloudflare Captcha bullshit.

I am running through TOR, but I'd assume cloudflare should present the captcha on access to `/`, rather then allow everything but the `/static/fonts/roboto/*` paths.

To make things extra confusing, if I manually navigate to the font URLs, they work properly.

TL;DR cloudflare is actively ruining the internet.

------

Edit: Never mind about index page render failure, I derped a noscript.

**Comment by kubotah** [comment]
>Edit: Never mind about load failure, I derped a noscript.

So can this be closed?

**Comment by herp-a-derp** [comment]
No, sorry, you didn't see the original report. I initially thought it was breaking the root page rendering, but that was me being a derp about noscript.

**Comment by kubotah** [comment]
That is strange, I could not reproduce. Perhaps because Cloudflare only blocks certain Tor nodes?

I just added a page rule to disable security for `https://yiff.party/static/fonts/`. Can you please check if the issue is fixed?

**Comment by herp-a-derp** [comment]
They're still 403-ing. Huh.

I refreshed my TOR circuit, got a cloudflare check page on `/`, and after stepping through that, **one* of the font files failed to load, but the other loaded file. I have no fucking idea what's going on.

For the most part, it's not really a problem, per-se. I'd pretty much just say cloudflare makes using TOR miserable, and move on. I bet they're trying to do something "smart" like their HTML parsing thing that lead to cloudbleed, and fucking that up royally again.
